### DISCUSSION TITLE
Sharepoint PNP module is getting auto updated to  newer versions

### DISCUSSION THREAD

**Discussion by Ramyaaarav** [pnp/powershell#1980] [Q&A]
### Discussed in https://github.com/pnp/powershell/discussions/1977

<div type='discussions-op-text'>

<sup>Originally posted by **Ramyaaarav** June 14, 2022</sup>
Hi All,

We have robotic process automaiton(RPA) uses cases which uses sharepoint pnp modue to launch sharepoint sites. we use the version 3.10.1906.0 of SharePointPnPPowerShellOnline which is tested and used by our bots. BUt recently observing the version is getting auto updated in azure windows virtual machines....</div>

**Comment by veronicageek** [comment]
@Ramyaaarav - This version/module of PnP PowerShell (`3.10.1906.0` of `SharePointPnPPowerShellOnline`) is archived and not maintained anymore. You should use the (new) PnP PowerShell (this repo) if you can. I'll transfer this back to discussions.

**Comment by Ramyaaarav** [comment]
How pnp powershell and SharePointPnPPowerShellOnline are diffrent

**Comment by AndersRask** [comment]
SharePointPnPPowerShellOnline was deprecated years ago and is no longer being updated.
You can read here how to upgrade
https://pnp.github.io/powershell/articles/upgrading.html
